keys/david.wallace.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCivybuw3qcfpjCeZJovOa4jHKdzDgzyyNoRNjxY4YCjbwEwkUGOMSoHvToZTjqctweXYPwgNxOFXqs0xlUsZWE43aMNamvID+J6Zj1aWsil+uBBJDkh1UWBz9SsuCekgpAyCUbYrnM+KvwZMSXNyRlDsxEjvS54DnDbIC9cJ3gOlnQ2LJB06BQSsWlyxMcsdXhBlze5e5sxepGlRoxDtjY9Cg0usTURPyvPFTpjtCCzzE4hy9GP+tBS4gzW4PmNlYfsWzDpXD0nFoG0NFwsKlGFAf6gZMusrb1rgJHP+ljZf630HkLtOs47sBQO09jWHVZOWv2YRDWyuDONntfREHMvVgX8otbJ7VFuFGxdPs7X1KicB8lJ1eIZ8EU1dRN3HnHaLXKO7YP9RDQPqpJzxza7qZbZ2bIboK7jMxEqyhnbaXFTPTy2ehBdCg1jyx6DZYq8XDeyxeJzu3m2xe92T3807bwrk9MF9HlUQ+VkCsqn9L47YfuVeUxhXiHnsjUCYnmUtJIJmwNYt/evYS4Bjhbd7Ungr0HCjB9+mcceBfr1LIDK2CZMbOs0akS0qRRUZ286rXtGr8U3EVfcxS+TuAWkIea1Xwk8WWUrjOHBS92wGM8D2TvcatrYPcqm4Mqi6PSqJRkXayB/SJcU2658AMkmq5acYX7ZpOll1uJ2Hg8oQ== david.wallace@goodeggs.com
```

Comment:
since you're all yubikey'd up, you should commit _that_ ssh key.  once it's all working, you should be able to type `ssh-add -L` to see it, and it'll end in `cardno: xxxx` instead of an email address.

Reply:
@bobzoller oh, interesting. Thanks for the heads up. Should be updated now. Out of curiosity, is not everyone required to the YubiKey SSH Key? I see plenty of other ssh keys in the `keys/` directory that don't seem to use it.

Reply:
nope, something we're moving _towards_, and holding devops folks to so far